src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCasePreviewServiceTest.java

```
@@ -1742,6 +1756,192 @@ public void givenCaseWithFirstWithInvalidDateTime_thenDisplayAnErrorAndDoNotDisp
+        NoticeIssuedTemplateBody templateBody = verifyTemplateBody(NoticeIssuedTemplateBody.ENGLISH_IMAGE, "Appellant Lastname", "face to face hearing", true);
+        assertEquals("a sign language interpreter (Sign (American Sign))", templateBody.getAdjournCaseTemplateBody().getInterpreterDescription());
+    }
+
+    @Test
+    public void givenAValidNonSignLanguageKey_ThenExtractTheInterpreterDescription(){
+        commonSetupForInterpreterDescription(sscsCaseData);
+        DynamicListItem item = new DynamicListItem("senegalFrench", "Senegal (French) Olof Dialect");
+        sscsCaseData.setAdjournCaseInterpreterLanguage(new DynamicList(item, Arrays.asList()));
+        final PreSubmitCallbackResponse<SscsCaseData> response = service.preview(callback, DocumentType.DRAFT_ADJOURNMENT_NOTICE, USER_AUTHORISATION, true);
+        NoticeIssuedTemplateBody templateBody = verifyTemplateBody(NoticeIssuedTemplateBody.ENGLISH_IMAGE, "Appellant Lastname", "face to face hearing", true);
+        assertEquals("a sign language interpreter (Senegal (French) Olof Dialect)", templateBody.getAdjournCaseTemplateBody().getInterpreterDescription());
+    }
+
+    @Test
+    public void givenAnInvalidLanguageKey_ThenReturnNullInterpreterDescription(){
+        commonSetupForInterpreterDescription(sscsCaseData);
+        DynamicListItem item = new DynamicListItem("something", null);
+        sscsCaseData.setAdjournCaseInterpreterLanguage(new DynamicList(item, Arrays.asList()));
+        final PreSubmitCallbackResponse<SscsCaseData> response = service.preview(callback, DocumentType.DRAFT_ADJOURNMENT_NOTICE, USER_AUTHORISATION, true);
+        NoticeIssuedTemplateBody templateBody = verifyTemplateBody(NoticeIssuedTemplateBody.ENGLISH_IMAGE, "Appellant Lastname", "face to face hearing", true);
+        assertEquals("a sign language interpreter (null)", templateBody.getAdjournCaseTemplateBody().getInterpreterDescription());
+    }
```

THis should throw an exception not return "a sign language interpreter (null)"